README.md

```diff
@@ -50,6 +52,8 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
 [discord-url]: https://discord.gg/rTKhrqz
 [heroku-img]: https://www.herokucdn.com/deploy/button.png
 [heroku-url]: https://heroku.com/deploy?template=https://github.com/nightscout/cgm-remote-monitor
+[railway-img]: https://railway.app/button.svg
+[railway-url]:https://railway.app/new/template?template=https%3A%2F%2Fgithub.com%2Fnightscout%2Fcgm-remote-monitor&plugins=mongodb&envs=API_SECRET%2CAUTH_DEFAULT_ROLES%2Cenable%2CMONGODB_URI%2CDISPLAY_UNITS&optionalEnvs=MONGODB_URI&API_SECRETDesc=A+passphrase+that+must+be+at+least+12+characters+long.&AUTH_DEFAULT_ROLESDesc=Possible+values+readable%2C+denied%2C+or+any+valid+role+name.+When+readable%2C+anyone+can+view+Nightscout+without+a+token.+Setting+it+to+denied+will+require+a+token+from+every+visit%2C+using+status-only+will+enable+api-secret+based+login.&enableDesc=Plugins+to+enable+for+your+site.+Must+be+a+space-delimited%2C+lower-case+list.&MONGODB_URIDesc=The+connection+string+required+to+access+your+database+%28where+all+your+data+is+stored+in+the+cloud%29.&DISPLAY_UNITSDesc=Preferred+BG+units+for+the+site%3A+mg%2Fdl+or+mmol%2FL+%28or+just+mmol%29.+Setting+to+mmol%2FL+puts+the+entire+server+into+mmol%2FL+mode+by+default%2C+no+further+settings+needed.&AUTH_DEFAULT_ROLESDefault=denied&enableDefault=careportal+basal+dbsize+rawbg+iob+maker+cob+bwp+cage+iage+sage+boluscalc+pushover+treatmentnotify+loop+pump+profile+food+openaps+bage+alexa+override+speech+cors&referralCode=9Y13IY
```

Comment:
referralCode=9Y13IY ?

Reply:
Hi - yes, i added my referral code, as mentioned in the initial comment [here](https://github.com/nightscout/cgm-remote-monitor/pull/7319#issue-1122666169). I'm perfectly fine with removing it though :)